test/kubernetes_unittest.cc

```diff
@@ -243,16 +243,16 @@ TEST_F(KubernetesTest, GetPodMetadata) {
   json::value expected_metadata = json::object({
     {"blobs", json::object({
       {"api", json::object({
+        {"version", json::string("1.6")},
         {"raw", json::object({
           {"metadata", json::object({
             {"creationTimestamp",
-              json::string("2018-03-03T01:23:45.678901234Z")},
+             json::string("2018-03-03T01:23:45.678901234Z")},
```

Comment:
Do we have a format checker or plan to have one?

Reply:
Good idea, but let's open a ticket to do this as a future improvement.